README.md

```diff
@@ -54,275 +55,250 @@ limitations under the License.
 </h4>
 
 <h3 align="center">
-    <p>State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow</p>
+    <p>State-of-the-art pretrained models for inference and training</p>
```

Comment:
Removed mention of TF/JAX since it seems like we're kinda moving away from it. Let me know if you want me to keep it!